client/status/common.go

```diff
@@ -51,13 +51,13 @@ func (x ServerInternal) Is(target error) bool {
 	}
 }
 
-// implements local interface defined in FromStatusV2 func.
+// implements local interface defined in ErrorFromV2 func.
 func (x *ServerInternal) fromStatusV2(st *status.Status) {
 	x.v2 = *st
 }
 
 // ToStatusV2 implements StatusV2 interface method.
-// If the value was returned by FromStatusV2, returns the source message.
+// If the value was returned by ErrorFromV2, returns the source message.
```

Comment:
If we're touching this then I'd expect `[ErrorFromV2]` to have less things to worry about in #409.

Reply:
documentation updated